doc/release-notes/release-notes-0.11.2.md

```diff
@@ -65,7 +62,13 @@ darkcoind/darkcoin-qt (on Linux).
 0.11.2 changelog
 ----------------
 
-- *to do ..*
+- Masternode Blinding
+- New Masternode Management Class
+- Masternode cache, when restarting the client the masternodes will be stored in the datadir
```

Comment:
"... in mncache.dat file (make sure you don't have any wallet with such name or this function will not be available for you otherwise"